cmd/action.go

```diff
@@ -61,7 +61,7 @@ func NewActx(cmd *cobra.Command, args []string) (ActionCtx, error) {
 	s, err := GetStore()
 	// in the case of add operator, there might not be a store
 	if err == ErrNoOperator {
-		if cmd.Name() == "operator" && cmd.Parent().Name() == "add" {
+		if cmd.Name() == "load" || (cmd.Name() == "operator" && cmd.Parent().Name() == "add") {
```

Comment:
let's not change this, and simply add the "load" command as a top-level command that is not exposed.

Reply:
ok taking a look

Reply:
change to `action` was not needed so took out this change